Model/final_project.py

```
@@ -0,0 +1,196 @@
+
+Automatically generated by Colab.
+
+Original file is located at
+    https://colab.research.google.com/drive/1KP92gjlVDWp_YKdKsk7M5Cp11H9UE4kv
+"""
+
+from google.colab import files
+uploaded = files.upload()
+
+import pandas as pd
+import numpy as np
```

Ruff (0.12.2)

`numpy` imported but unused

Remove unused import: `numpy`

(F401)